src/config/index.ts

```diff
@@ -0,0 +1,10 @@
+const configuration = {
+	aws_project_region: '',
+	aws_cognito_identity_pool_id: '',
+	aws_cognito_region: '',
+	aws_user_pools_id: '',
+	aws_user_pools_web_client_id: '',
+	oauth: '',
+}
+
+export default configuration
```

Comment:
This is useless. There needs to be process how to populate this configuration from AWS. Let's try it together later. :wink:

Reply:
Ok, I delete the configuration settings for now. Due to pipeline. I yand import configuration from gitignored json settings